src/components/EditPostModal/index.tsx

```diff
@@ -257,13 +267,22 @@ const EditPostModal = ({
             keyboardVerticalOffset={Platform.select({ ios: 100, android: 80 })}
             style={styles.AllInputWrap}
           >
-            <ScrollView style={styles.container}>
-              <TextInput
-                multiline
-                placeholder="What's going on..."
+            <ScrollView
+              style={styles.container}
+              keyboardShouldPersistTaps="handled"
+            >
+              <MentionInput
                 style={styles.textInput}
-                value={inputMessage}
-                onChangeText={(text) => setInputMessage(text)}
+                isBottomMentionSuggestionsRender={true}
+                placeholder="What's going on...?"
```

Comment:
Do we have i18n ?

Reply:
We had already installed that package. But not setup properly yet.